src/components/Search.js

```diff
@@ -16,7 +17,8 @@ export const Search = () => {
   const inputRef = useRef()
   const listRef = useRef()
   const i18n = useI18n()
-  const [current, setNavigation, getCurrent] = useNavigation('Search', containerRef, listRef, 'y')
+  const [isFeedExpanded, setIsFeedExpanded] = useState(false)
+  const [current, setNavigation, getCurrent, getAllElements, navigateNext, navigatePrevious] = useNavigation('Search', containerRef, listRef, 'y')
```

Comment:
just looking at the source, there is a way to cleaning some codes here

`navigateNext` is also `setNavigation(getCurrent().index + 1)`
and 
`navigatePrevious` is also `setNavigation(getCurrent().index - 1)`